ChangeLog

```diff
@@ -1,6 +1,7 @@
 CHANGES
 =======
 
+ADD: rename getToken2 to getTokenByDomainId
```

Comment:
```suggestion
```

Internal refactor. I think it shouldn't be mentioned in Changelog (which use to have an "end user" perspective)